bin/generate-html.js

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env node
+"use strict";
+const defaults = {
+  baseUrl: "./"
+};
+
+function template(rawOptions) {
+  const options = Object.assign({}, defaults, rawOptions || {});
```

Comment:
Not sure if you want to just default the `rawOptions` on L7 to have a default value of `{}` and then you probably wouldn't need the `|| {}` here.